packages/client/src/pages/projects/NewProject.tsx

```diff
@@ -59,13 +62,22 @@ export const NewProject: React.FC = () => {
       setError(false);
     }
   };
-  const handleSubmit = (event: { preventDefault: () => void }) => {
+  const handleSubmit = async (event: { preventDefault: () => void }) => {
     if (error) {
       setError(true);
       event.preventDefault();
       return;
     } else {
-      //submit logic
+      const result = await apolloClient.mutate({
```

Comment:
Its best to leverage the generated React hooks since they come with additional type checking, generate the Apollo Client is only used directly if a series of requests need to happen in order.

Something I just found out is you can specify the return type via
```
apolloClient.mutate<Project>({ ... })
```